github/repos_test.go

```diff
@@ -623,3 +623,55 @@ func TestRepositoriesService_License(t *testing.T) {
 		t.Errorf("Repositories.License returned %+v, want %+v", got, want)
 	}
 }
+
+func TestRepositoriesService_GetRequiredStatusChecks(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/branches/b/protection/required_status_checks", func(w http.ResponseWriter, r *http.Request) {
+		v := new(ProtectionRequest)
+		json.NewDecoder(r.Body).Decode(v)
+
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeProtectedBranchesPreview)
+		fmt.Fprint(w, `{"include_admins": true,"strict": true,"contexts": ["x","y","z"]}`)
+	})
+
+	contexts, _, err := client.Repositories.GetRequiredStatusChecks(context.Background(), "o", "r", "b")
+	if err != nil {
+		t.Errorf("Repositories.ListRequiredStatusChecksContexts returned error: %v", err)
```

Comment:
It looks like you've made some copy paste errors here, this should be `GetRequiredStatusChecks`, not `ListRequiredStatusChecksContexts`. Here, and below. Also, the returned value is better named `checks`, not `contexts`.

This is just a minor naming/error message issue in the tests, I can fix this while merging.